nomenclature/codelist.py

```diff
@@ -624,3 +624,19 @@ def filter(self, hierarchy: str) -> "RegionCodeList":
                 "Use `RegionCodeList.hierarchy` method for available items."
             )
             raise ValueError(msg)
+
+
+class MetaCodeList(CodeList):
+    """A subclass of CodeList specified for MetaCodes
+
+    Attributes
+    ----------
+    name : str
+        Name of the MetaCodeList
+    mapping : dict
+        Dictionary of `MetaCode` objects
+
+    """
+
+    code_basis: ClassVar = MetaCode
+    validation_schema: ClassVar[str] = "meta"
```

Comment:
In conjunction with the push to remove all the schemas I'd leave this out and just use have `DataStructureDefinition` use the generic schema.
```suggestion
```